pages/api/event.ics.js

```diff
@@ -0,0 +1,66 @@
+import { createEvent } from "ics";
+
+function parseDateString(dateString) {
+  const parts = dateString.split(" ");
+
+  const dateParts = parts[0].split("-");
+  const month = parseInt(dateParts[0], 10) - 1;
+  const day = parseInt(dateParts[1], 10);
+  const year = parseInt(dateParts[2], 10);
+
+  const timeParts = parts[1].toLowerCase().split(":");
+  let hours = parseInt(timeParts[0], 10);
+  const minutes = parseInt(timeParts[1], 10);
+
+  if (parts[2] === "pm") {
+    hours += 12;
+  }
+
+  return new Date(year, month, day, hours, minutes);
+}
+let alarms = [
+  {
+    action: "audio",
+    description: "Reminder",
+    trigger: { hours: 0, minutes: 5, before: true },
+    repeat: 2,
+    attachType: "VALUE=URI",
+    attach: "Glass",
+  },
+];
+export default function handler(request, response) {
+  if (request.method == "GET") {
+    const title = request.query.title;
```

Comment:
We could DRY this up a bit by restructuring the properties from request.query.

e.g. 

``` js 
// const title = request.query.title;
// const start = request.query.start;
// const duration = request.query.duration;
// const description = request.query.description;

const { title, start, duration, description } = request.query;
```